code/modules/mob/living/simple_animal/vore/rous.dm

```diff
@@ -43,4 +43,117 @@
 
 /mob/living/simple_animal/hostile/rous/death()
 	playsound(src, 'sound/effects/mouse_squeak_loud.ogg', 50, 1)
+	if(resting)
+		resting = 0
+		icon_state = icon_dead
+		update_icon()
 	..()
+
+/mob/living/simple_animal/hostile/rous/passive
+	name = "curious giant rat"
+	desc = "In what passes for a hierarchy among verminous rodents, this one is king. It seems to be more interested on scavenging."
+	follow_dist = 1
+	var/mob/living/carbon/human/food
+	var/hanger = 0
+
+/mob/living/simple_animal/hostile/rous/passive/Life()
+	. = ..()
+	if(!. || ai_inactive) return
+
+	for(var/obj/item/weapon/reagent_containers/food/snacks/S in oview(src,7)) //Accept thrown offerings and scavenge surroundings.
+		if(isturf(S.loc))
+			if(get_dist(src,S) <=1)
+				visible_emote("hungrily devours \the [S].")
+				playsound(src.loc,'sound/items/eatfood.ogg', rand(10,50), 1)
+				qdel(S)
+				hanger = 0
+				food = null
+			else
+				WanderTowards(S.loc)
+			break
+
+	if(!food) return
+
+	var/food_dist = get_dist(src,food)
+
+	if(food_dist > 7) //Lose interest on this person.
+		food = null
+		if(hanger < 30) //Get miffed but don't let people evade stack you into pouncing the next victim on sight.
+			hanger += 5
+
+	if(food_dist > 1)
+		if(stance == STANCE_IDLE)
+			if(set_follow(food,10 SECONDS))
+				handle_stance(STANCE_FOLLOW)
+				if(resting)
+					lay_down()
+
+	if(food_dist <= 1)
+		if(hanger < 15)
+			if(prob(25))
+				visible_emote(pick("sniffs curiously at [food].",
+								   "stares at [food], seeming to want something.",
+								   "sniffs at [food]'s hands.",
+								   "sniffs curiously at [food]'s pockets.",
+								   "sits down for a moment, reaching towards [food] with its paws."))
+				hanger += 5
+		else if(hanger < 30)
+			if(prob(25))
+				visible_emote(pick("sniffs intently against [food], especially their pockets and gear.",
+								   "stands up to beg [food] for snacks.",
+								   "attempts to burrow into [food]'s pockets.",
+								   "leans against [food], licking its chops.",
+								   "hungrily nibbles onto [food]."))
+				hanger += 5
+		else if(hanger < 45)
+			if(prob(25))
+				visible_emote(pick("growls at [food], sounding rather hangry!",
+								   "aggressively bumps and nudges against [food], trying to make something fall out.",
+								   "salivates at [food] in an unsettling manner.",
+								   "pushes hard against [food], licking its chops.",
+								   "almost sinks its teeth into [food], just stopping to give them another chance."))
+				hanger += 5
+		else if(hanger < 50)
+			visible_emote("appears to have had enough and prepares to strike!")
+		else
```

Comment:
I feel like there is probably a better way of doing this last bit, as I think it's handled in the attack() proc, but I could be wrong.

Reply:
It's sort of a passive state of following and begging. I feel like putting it into attack proc would mess things up what comes to regular combat against these.